awx/main/utils/licensing.py

```diff
@@ -383,12 +384,24 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        license_date = int(attrs.get('license_date', 0) or 0)
+        if HostMetric:
+            automated_instances = HostMetric.objects.count()
+            first_host = HostMetric.objects.order_by('first_automation').first()
```

Comment:
```    "main_hostmetric_pkey" PRIMARY KEY, btree (hostname)
    "main_hostmetric_first_automation_b747a0a1" btree (first_automation)
    "main_hostmetric_hostname_87ac3c1f_like" btree (hostname varchar_pattern_ops)
    "main_hostmetric_last_automation_11010683" btree (last_automation)
``` Index exists on `first_automation`, so this is great.

Reply:
Updated